monkey/infection_monkey/telemetry/scoutsuite_telem.py

```diff
@@ -1,11 +1,12 @@
 from common.cloud.scoutsuite.ScoutSuite.output.result_encoder import ScoutJsonEncoder
+from common.cloud.scoutsuite.ScoutSuite.providers.aws.provider import AWSProvider
 from common.common_consts.telem_categories import TelemCategoryEnum
 from infection_monkey.telemetry.base_telem import BaseTelem
 
 
 class ScoutSuiteTelem(BaseTelem):
 
-    def __init__(self, data):
+    def __init__(self, data: AWSProvider):
```

Comment:
Something doesn't make sense if you have a var called `data` of type `AWSProvider`

Reply:
True